tests/cypress/e2e/actions_tasks3/case_105_cloud_storage.js

```diff
@@ -33,7 +33,7 @@ context('Cloud storage.', () => {
     describe(`Testing case "${caseId}"`, () => {
         it('Check "Cloud Storage" page.', () => {
             cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
-            cy.get('.cvat-empty-cloud-storages-list').should('be.visible');
+            cy.get('.cvat-empty-cloud-storages-list > .ant-empty').should('be.visible');
```

Comment:
no the .cvat-empty-cloud-storages-list component doesnt has any width or height so it is undetectable and breaks the tests.

Reply:
Ok, then will `exist` work?
```
cy.get('.cvat-empty-cloud-storages-list').should('exist');
```
Its better to avoid ant selectors where possible, it makes migration to new ant versions simplier.